test/plots/barPlotTests.ts

```diff
@@ -773,6 +773,20 @@ describe("Plots", () => {
         assert.include(["visible", "inherit"], label2.style("visibility"), "label 1 is visible");
         svg.remove();
       });
+
+      it("hides labels that are partially cut off in y", () => {
+        yScale.domain([1, 2]);
+        let texts = svg.selectAll("text");
```

Comment:
Instead of selecting inside the `svg`, please select inside of `barPlot.content()`.  It's just a lot more precise I feel.